### PR TITLE
[DOCS-6308] Display currently selected version on product landing pages

### PR DESCRIPTION
**Pull request by gravitonian**
Note that some the products don't display a version drop-down selector in the Docs site, but have version "1.0" in the product title. In these cases, the content always reflects the latest release of that product:

- Alfresco Process Automation 1.0
- Alfresco Mobile Workspace 1.0
- Alfresco Content Services for Mobile 1.0
- Alfresco Support Handbook 1.0

### CODE REVIEW

File: _layouts/docs.html [hunk 1]
```diff
@@ -42,6 +41,13 @@
             <button class="button is-rounded" id="table-of-contents" onclick="tocToggler()">Table of contents</button>
             <div class="content">
                 <h1>{{page.title}}</h1>
```
Comment (anxumalo) [inline _layouts/docs.html:43]: This looks good @gravitonian but I wonder if we can move the version number after the title to help improve search hits (as shown below)? 

```suggestion
                <h1>{{page.title}} {{page.version}}</h1>
```

It's also a really elegant solution as it updates the template, and saves us having to update every product landing page. However, we also need to be more flexible as some products don't display a version number (e.g. APA - on PaaS, all Mobile docs, Support Handbook):

- https://docs.alfresco.com/process-automation/latest/
- https://docs.alfresco.com/content-mobile/latest/

Can we exclude the version number if `versions` are not defined in `_config.yml`?

Comment (gravitonian) [reply _layouts/docs.html:43]: Versions are specified in _config.yml, the above products (process-automation and content-mobile) displays version 1.0 as specified in _config.yml.

Comment (gravitonian) [reply _layouts/docs.html]: Version is now in the product title

**Comment by gravitonian** [comment]
Created a branch for this one that has the wrong DOCS ticket number, changed the PR title to reflect the correct DOCS ticket

**Review by anxumalo — COMMENTED** [review]
Almost there! Can we exclude the Community versions, so we can avoid titles like:

- Alfresco Governance Services Community Edition community
- Alfresco Community Edition community

I think it's worth noting in the PR summary that these products are currently displayed as version "1.0". So we don't display a version drop-down selector for the following products as the content always reflects the latest release:

- Alfresco Process Automation 1.0
- Alfresco Mobile Workspace 1.0
- Alfresco Content Services for Mobile 1.0
- Alfresco Support Handbook 1.0

File: _layouts/docs.html [hunk 2]
```diff
@@ -41,7 +40,15 @@
...
-                <h1>{{page.title}}</h1>
+              <!-- If it is the landing page for a product, then print out the currently selected version for the product. -->
+              <!-- A page is considered to be the landing page if the path to it is <product id>/<product version>/index.md -->
+              <!-- Any longer path is considered a sub page, which should not have the product version printed out -->
+              {% assign pathElements = page.path | split: "/"%}
+              {% if pathElements.size == 3 %}
+              <h1>{{page.title}} {{page.version}}</h1>
+              {% else %}
+              <h1>{{page.title}}</h1>
+              {% endif %}
```
Comment (anxumalo) [inline _layouts/docs.html:51]: Maybe you can find a more elegant way of doing this? Tested locally and it works.

```suggestion
              <!-- For each landing page, print out the selected version for the product (unless it's a Community version). -->
              <!-- A landing page has a path with format: <product-name>/<product-version>/index.md -->
              <!-- A longer path is considered a sub page, which should not have the version printed out. -->
              {% assign pathElements = page.path | split: "/"%}
              {% if pathElements.size == 3 %}
              {% unless page.version == 'community' %}
              <h1>{{page.title}} {{page.version}}</h1>
              {% else %}
              <h1>{{page.title}}</h1>
              {% endunless %}
              {% else %}
              <h1>{{page.title}}</h1>
              {% endif %}
```

**Review by anxumalo — APPROVED** [review]
LGTM 👍🏼

**Pull request merged**